kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h

```diff
@@ -654,24 +613,15 @@ class ResidualBasedBlockBuilderAndSolver
         BaseType::mDofSet = Doftemp;
 
         //Throws an exception if there are no Degrees Of Freedom involved in the analysis
-        if (BaseType::mDofSet.size() == 0)
-        {
-            KRATOS_THROW_ERROR(std::logic_error, "No degrees of freedom!", "");
-        }
-        if( this->GetEchoLevel() > 2)
-        {
-            KRATOS_WATCH(BaseType::mDofSet.size())
-        }
+        KRATOS_ERROR_IF(BaseType::mDofSet.size() == 0) << "No degrees of freedom!" << std::endl;
+
+        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", ( this->GetEchoLevel() > 2)) << "Number of degrees of freedom:" << BaseType::mDofSet.size() << std::endl;
+
         BaseType::mDofSetIsInitialized = true;
-        if( this->GetEchoLevel() > 2 && r_model_part.GetCommunicator().MyPID() == 0)
-        {
-            std::cout << "Finished setting up the dofs" << std::endl;
-        }
 
-        if( this->GetEchoLevel() > 2)
-        {
-            KRATOS_WATCH("Initializing lock array")
-        }
+        KRATOS_INFO_IF("ResidualBasedBlockBuilderAndSolver", ( this->GetEchoLevel() > 2 && r_model_part.GetCommunicator().MyPID() == 0)) << "Finished setting up the dofs" << std::endl;

```

Comment:
the if should be the same in the two cases (i mean, it would make sense if it was the same)

Reply:
I use the same level was defined previously